runtime/jcl/unix/syshelp.c

```diff
@@ -222,15 +204,17 @@ jobject getPlatformPropertyList(JNIEnv * env, const char *strings[], int propInd
 	}
 
 	plist = createSystemPropertyList(env, strings, propIndex);
-	if (envSpace)
+	if (NULL != envSpace) {
 		jclmem_free_memory(env,envSpace);
-	if (homeAlloc) jclmem_free_memory(env, homeAlloc);
+	}
+
+failed:
+	if (NULL != homeAlloc) {
+		jclmem_free_memory(env, homeAlloc);
+	}
 	return plist;
 }
 
-#undef J9PWENT
```

Comment:
is this macro used anywhere? is it enabled by default?

Reply:
https://github.com/eclipse-openj9/openj9/blob/548cace9ce647425c64fe08ce9d1c3ebc4284dff/runtime/jcl/unix/syshelp.c#L44-L46
It can be removed along with the obsoleted define `CHORUS`